mkdocs.yml

```diff
@@ -145,13 +145,14 @@ nav:
             - 'Shop configuration': 'guide/shop_configuration.md'
         - 'Admin management': 'guide/admin_panel.md'
         - Content rendering:
-            - 'Content rendering': 'guide/content_rendering.md'
-            - 'Controllers': 'guide/controllers.md'
-            - 'Templates': 'guide/templates.md'
-            - 'Design engine': 'guide/design_engine.md'
-            - 'Page rendering': 'guide/page_rendering.md'
-            - 'Displaying Content children': 'guide/displaying_children_of_a_content_item.md'
-            - 'Retrieving root Location': 'guide/retrieving_root_location.md'
+            - Render content:
+                - 'Render content': 'guide/content_rendering/render_content/render_content.md'
+                - 'Render page': 'guide/content_rendering/render_content/render_page.md'
+            - Template configuration and matching:
+                - 'Template configuration': 'guide/content_rendering/templates/template_configuration.md'
```

Comment:
Maybe it would be good to switch positions of Template configuration and Templates to have Templates first, then Template configuration?